src/OrleansServiceFabricUtils/OrleansCommunicationListener.cs

```diff
@@ -62,7 +63,7 @@ public OrleansCommunicationListener(ServiceContext context, ClusterConfiguration
             var endpoints = activation.GetEndpoints();
             var siloEndpoint = GetEndpoint(endpoints, SiloEndpointName);
             var gatewayEndpoint = GetEndpoint(endpoints, GatewayEndpointName);
-
+            this.SiloHost = new SiloHostWrapper();
```

Comment:
why this change?

Reply:
I think it is better to init those fields in the constructor, instead of init those directly from the field.